src/Microsoft.AspNet.Authentication.OAuth/Events/OAuthCreatingTicketContext.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNet.Authentication.OAuth
     /// <summary>
     /// Contains information about the login session as well as the user <see cref="System.Security.Claims.ClaimsIdentity"/>.
     /// </summary>
-    public class OAuthCreatingTicketContext : BaseContext<OAuthOptions>
+    public class OAuthCreatingTicketContext : BaseContext
```

Comment:
I know, it was a tradeoff with getting rid of generics which let us somewhat cleanly have the events derive from RemoteAuthenticationEvents